desktop/hardened-desktop/registry-access-management.md

```diff
@@ -12,11 +12,13 @@ redirect_from:
 
 With Registry Access Management, administrators can ensure that their developers using Docker Desktop only access registries that are allowed. This is done through the Registry Access Management dashboard on Docker Hub. 
 
-Below are some example registries administrators can allow: 
+RAM supports both cloud and on-prem registries. Example registries administrators can allow include: 
```

Comment:
Suggest we use the full form Registry Access Management here. The Hub UI also doesn't use the acronym.